src/equations/compressible_euler_2d.jl

```diff
@@ -1263,4 +1263,43 @@ end
 end
 
 
+# Calculate the vorticity on a single node using the derivative matrix from the polynomial basis of
+# a DGSEM solver.
+# !!! Note that this only works for DGSEM solvers on a TreeMesh !!!
```

Comment:
Pass the `mesh` and restrict it accordingly?

Reply:
Does not work, because `equations.jl` is included before `meshes.jl`.

Reply:
This seems to be a clear sign that this shouldn't live in `equations`. Since it is very specific to your new setup, I would prefer to move it somewhere else. We had the equations cluttered with solver-specific stuff before and were happy to get rid of that... What do you think, @sloede?

Reply:
`equations` was my suggestion for a lack of a better place where a function like this could live. Do you have a better suggestion - I am really open to anything!

Reply:
Looking at `src/solvers/dgmulti/flux_differencing_compressible_euler.jl`, it might be something for a new file `src/solvers/dgsem_tree/dg_2d_compressible_euler.jl` or something like that

Reply:
Sounds good to me - @lchristm can you give this a try and see if it resolves the cyclic redundancy issues?

Reply:
Created a separate file for the vorticity functions in f1e63b4 as @ranocha  suggested.